### ISSUE TITLE
Install elasticsearch needed in Readme installation instructions?

### ISSUE THREAD

**Issue by anselmbradford** [cfpb/owning-a-home#328]
Are ElasticSearch installation instructions needed in https://github.com/cfpb/owning-a-home#sheer--ElasticSearch in order to run OAH? After installation nothing is done with ElasticSearch.

**Comment by ascott1** [comment]
No, not really. Eventually they *should* be... but we're not using Sheer as anything other than as a server for static sites.

**Comment by cfarm** [comment]
This is needed now
